tests/foreman/ui/test_rhcloud_inventory.py

```diff
@@ -158,9 +160,7 @@ def test_obfuscate_host_names(
         session.organization.select(org_name=org.name)
         session.location.select(loc_name=DEFAULT_LOC)
         # Enable obfuscate_hostnames setting on inventory page.
-        session.cloudinventory.update({'obfuscate_hostnames': True})
```

Comment:
Any reason why its removed ?

Reply:
I believe this is another debugging artifact that I failed to remove. I'll revert the change.

Reply:
Deleting a line is a debugging artifact? That's fishy.